resources/user-agents/browsers/samsung/samsung-android-browser-2-0-to-6-0.json

```diff
@@ -53,6 +53,17 @@
             "Android_6_0"
           ]
         },
+        {
+          "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*) applewebkit* (*khtml*like*gecko*) SamsungBrowser/#MAJORVER#.#MINORVER#*Chrome*Safari*",
+          "devices": {
+            "SM-J320F*": "Samsung SM-J320F"
+          },
+          "platforms": [
+            "Android_7_1",
```

Comment:
This should be moved up above the ones that have `Android_7_0` as the highest android version.

Reply:
done